docs/guide/essentials/config/auto-imports.md

```diff
@@ -110,3 +102,17 @@ export default defineConfig({
   imports: false, // [!code ++]
 });
 ```
+
+## Explicit Imports
+
+Anything that is auto-imported can also be imported manually via `#imports`:
+
+```ts
+import {
+  createShadowRootUi,
+  ContentScriptContext,
+  MatchPattern,
+} from '#imports';
+```
+
+This is the preferred way of importing WXT APIs, but it's only available if auto-imports are enabled.
```

Comment:
Hmm, not very useful as a shorthand when auto-imports are enabled. only really useful if they're disabled. So that means I need to get this working when auto-imports are disabled.